### PR TITLE
Update example and reviewdog.yml

### PR DESCRIPTION
**Pull request by tsuyoshicho**
## example update

Use `github.workflow` , `steps.depup.outputs.repo` for Pull-request message.
This copy to use generically.

## depup.yml update

Update above change.

### CODE REVIEW

**Review by haya14busa — CHANGES_REQUESTED** [review]
How about using `steps.depup.outputs.repo` instead of `github.job`?

**Comment by tsuyoshicho** [comment]
> How about using `steps.depup.outputs.repo` instead of `github.job`?

I think to keep the characteristics of the old format.
update it.

File: .github/workflows/depup.yml [hunk 1]
```diff
@@ -46,11 +45,10 @@ jobs:
         uses: peter-evans/create-pull-request@v3
         with:
           token: ${{ secrets.GITHUB_TOKEN }}
-          title: "chore(deps): update redpen to ${{ steps.depup.outputs.latest }}"
-          commit-message: "chore(deps): update redpen to ${{ steps.depup.outputs.latest }}"
+          title: "chore(deps): update ${{ steps.depup.outputs.repo }} to ${{ steps.depup.outputs.latest }}"
+          commit-message: "chore(deps): update ${{ steps.depup.outputs.repo }} to ${{ steps.depup.outputs.latest }}"
           body: |
-            Update redpen to [${{ steps.depup.outputs.latest }}](https://github.com/redpen-cc/redpen/releases/tag/redpen-${{ steps.depup.outputs.latest }})
```
Comment (haya14busa) [inline .github/workflows/depup.yml:52]: `redpen-` prefix should be required.

Comment (tsuyoshicho) [reply .github/workflows/depup.yml:52]: oops... I check link, and fix it.

Comment (tsuyoshicho) [reply .github/workflows/depup.yml:52]: fixed

**Review by haya14busa — APPROVED** [review]
:+1:

**Pull request merged**